fastapi-google-stt/app/api/openai_router.py

```
@@ -0,0 +1,49 @@
+        - message: 번역할 메시지
+    """
+    print("Streaming started")
+    return StreamingResponse(
+        get_streaming_message_from_openai(data), media_type="text/event-stream"
+    )
+
+@router.post(
+    "/header-test",
+    summary="Generate messages using OpenAI (streaming) - header test",
+)
+async def get_generated_messages_streaming(
```

Ruff (0.8.2)

Redefinition of unused `get_generated_messages_streaming` from line 14

(F811)